src/index.css

```diff
@@ -8,8 +8,71 @@ html {
   font-family: 'Noto Sans JP', sans-serif;
 }
 
-body {
-  margin: 0;
-  background-color: #3f3f3f;
+
```

Comment:
You should be using styled components instead of plain CSS. I mentioned this to you when I assigned the task.

Reply:
Also add some style to your input field.